lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +227,29 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        for div in article_bs.find_all("div", {"class": "title"}, limit=24):
+            for link in div.select("a"):
+                url = link['href']
+        return self.url_pattern + url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        urls = []
+
+        for url in self.get_search_urls():
```

Comment:
how do you control number of articles to collect? they are defined in config

Reply:
done